Sources/Prototyper/Classes/Extensions/UIImage+OverlapImages.swift

```diff
@@ -6,6 +6,9 @@
 //
 
 import Foundation
+#if canImport(UIKit)
+import UIKit
```

Comment:
As far as I can see the Swift Package seems to build fine without the `#if canImport(UIKit)` statements across the code. It builds successfully using Xcode when selecting an iOS target. I would suggest to remove the if statements.